iac-extensions/arm/src/test/resources/checks/ElementsOrderCheckTest/decoratorCustomFirst.bicep

```diff
@@ -0,0 +1,7 @@
+@customDecorator()
+// Noncompliant@+1
```

Comment:
Please check at least once the message of the check by using the `{{}}` notation